src/_pytest/outcomes.py

```diff
@@ -49,18 +49,23 @@ class Failed(OutcomeException):
 class Exit(KeyboardInterrupt):
     """ raised for immediate program exits (no tracebacks/summaries)"""
 
-    def __init__(self, msg="unknown reason"):
+    def __init__(self, returncode=None, msg="unknown reason"):
```

Comment:
Please add `returncode` to the end of the parameter list to keep backward compatibility; technically users should not raise this explicitly, but it doesn't hurt to keep compatibility in this case.